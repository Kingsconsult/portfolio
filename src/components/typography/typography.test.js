import React from 'react';
import renderer from "react-test-renderer";
import Typography  from "./index";
 
import { ThemeProvider } from "styled-components";
const theme = {
  mode: "light"
};


it("typography renders correctly when called ", () => {
  const tree = renderer
    .create(
      <ThemeProvider theme={theme}>
            <Typography />
      </ThemeProvider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});