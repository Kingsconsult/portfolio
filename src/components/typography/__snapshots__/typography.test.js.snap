// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cards renders correctly when called  1`] = `
<h1
  className="sc-AxirZ iNZWKN"
  size="small"
/>
`;

exports[`typography renders correctly when called  1`] = `
<h1
  className="sc-AxirZ iNZWKN"
  size="small"
/>
`;
